### PR TITLE
Add support for tee-supplicant loadable plugins

### PR DESCRIPTION
**Pull request by DemesneGH**
Hi @mssun 
The loadable plugins framework is a new feature in OPTEE 3.13.0. It allows TA actively invoke a command defined in normal world plugins (which are compiled as .so dynlibs).See [this commit](https://github.com/OP-TEE/optee_client/commit/1e91cc797f72261fb03e7c2ce30944ff613ac07e).
In this commit I've updated:
1. In`optee-teec` `optee-utee` and `optee-utee-sys`:Wrap for related apis;
2. Add a new example `supp_plugin` for tee-supplicant plugins.The work logic is `host->TA->plugin`.Plugins are identified by uuid.

Best Regards,
Yuan

### CODE REVIEW

File: optee-utee/src/extension.rs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+use crate::{Error, Result, Uuid};
+use optee_utee_sys as raw;
+
+pub struct LoadablePlugin<'a> {
+    uuid: &'a Uuid
```
Comment (mssun) [inline optee-utee/src/extension.rs:5]: How about using `uuid: Uuid`. No need to have a reference.

File: optee-utee/src/extension.rs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+impl<'a> LoadablePlugin<'a> {
+    pub fn new(uuid: &'a Uuid) -> Self {
+        Self { uuid }
```
Comment (mssun) [inline optee-utee/src/extension.rs:10]: -> `Self { uuid: uuid.to_owned() }`

**Review by mssun — CHANGES_REQUESTED** [review]
- Can you also rebase your commits on the current master?
- Please add license header to all source code files (i.e., `*.rs`, `Makefile`, `*.sh`)

**Comment by DemesneGH** [comment]
New PR #29

**Pull request closed without merging**